<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QDialog" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit Calculator (by Stephank)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(255, 219, 139);
}</string>
  </property>
  <widget class="QPushButton" name="calculator">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 160, 122);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>&lt;-- Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showResult">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 160, 122);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>430</y>
     <width>40</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Руб.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>80</y>
     <width>130</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Сумма вклада:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>185</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>130</y>
     <width>130</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>% годовых</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultProfit">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>430</y>
     <width>240</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>430</y>
     <width>40</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Руб.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="depositAmount">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>30</y>
     <width>130</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>293</y>
     <width>205</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Капитализация процентов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>191</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Доход за вычетом налогов:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>196</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="day">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	background-color: rgb(59, 60, 54);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>День</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="month">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	background-color: rgb(59, 60, 54);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Месяц</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="year">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	background-color: rgb(59, 60, 54);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Год</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>130</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>480</y>
     <width>40</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>185</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Налог:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultPercent">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>80</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Срок размещения:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="yes">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	background-color: rgb(59, 60, 54);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Да</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="no">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
	background-color: rgb(59, 60, 54);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Нет</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taxRate">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>180</y>
     <width>130</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>165</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Налоговая ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>230</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Периодичность выплат:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultSum">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>480</y>
     <width>240</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>480</y>
     <width>40</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Руб.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>480</y>
     <width>191</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Сумма к концу срока:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPayment">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 160, 122);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="addLayout">
    <item>
     <widget class="QScrollArea" name="scrollPayment">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>447</width>
         <height>147</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Пополнения:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePayment">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 160, 122);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteWaste">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 160, 122);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addWaste">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>230</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 160, 122);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>135</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black; 
}</string>
   </property>
   <property name="text">
    <string>Частичные снятия:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="wasteLayout">
    <item>
     <widget class="QScrollArea" name="scrollWaste">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>447</width>
         <height>147</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="periodCombo">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>235</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgb(59, 60, 54);
	color: white;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
